client/src/components/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Spinner.styles.scss'
+
+import { SkeletonText, Spinner as PageSpinner } from '@chakra-ui/react'
+
+interface SpinnerProps {
+  type?: string
+  width: string | number
+  height: string | number
+}
+
+const Spinner = ({ type, width, height }: SpinnerProps): JSX.Element => {
+  return type === 'paragraph' ? (
+    <SkeletonText />
+  ) : (
+    <div className="spinner" style={{ width: `${width}`, height: `${height}` }}>
```

Comment:
Since we are migrating to chakra, the `div` should also be chakra and the properties are styled in this file, and thus we can remove the corresponding `scss` file.

Reply:
I believe we still have to wrap it in a `Center` or equivalent to center it.

Reply:
Yeah the alignment didn't work without `Center`, so I've left it in